src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -574,6 +586,20 @@ internal int ExecuteTool(string pathToTool, string responseFileCommands, string
             }
 
             return ExitCode;
+
+            // Construct the friendly name for the compilation. This does not need to be unique. Instead
+            // it's used by developers to understand what compilation is running on the server.
+            string GetRequestId()
```

Comment:
Shouldn't local functions start with lowercase letter?
```suggestion
            string getRequestId()
```

Reply:
Yep